src/inventory-service/pom.xml

```diff
@@ -9,7 +9,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.inventory</groupId>
     <artifactId>com.inventory.parent</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
```

Comment:
I don't think there's really any reason to bump snapshot versions; they are inherently mutable anyway, and you're not gonna have anything else depending on this, right?

Reply:
If you went full java, you'd have `1.2.3-SNAPSHOT`, then you'd do a release, which'd release `1.2.3`, then bump the version to `1.2.4-SNAPSHOT` in the poms. For your purposes here i'd just keep 1.0.0-SNAPSHOT.